mmdet/models/roi_heads/sparse_roi_head.py

```diff
@@ -120,8 +124,8 @@ def _bbox_forward(self, stage, x, rois, object_feats, img_metas):
         bbox_head = self.bbox_head[stage]
         bbox_feats = bbox_roi_extractor(x[:bbox_roi_extractor.num_inputs],
                                         rois)
-        cls_score, bbox_pred, object_feats = bbox_head(bbox_feats,
-                                                       object_feats)
+        cls_score, bbox_pred, object_feats, attn_feats = bbox_head(
```

Comment:
Need to be compatible with spare_rcnn.

Reply:
This code has no influence on the training process of sparse_rcnn.

Reply:
If I use sparse rcnn configuration, the output length of bbox_head should be 3 instead of 4.This should give an error.

Reply:
We have modified the implementation of ```DIIHead``` and make sure this module will always return four values no matter it is built by ```QueryInst``` or ```SparseR-CNN```

Reply:
OK